test/zdtm/static/mnt_ro_root.c

```diff
@@ -0,0 +1,25 @@
+#include <sys/mount.h>
+
+#include "zdtmtst.h"
+
+const char *test_doc = "Check if root mount remains read-only after c/r";
+const char *test_author = "Pavel Tikhomirov <ptikhomirov@virtuozzo.com>";
+
+char *dirname;
+TEST_OPTION(dirname, string, "directory name", 1);
+
+int main(int argc, char **argv)
+{
+	test_init(argc, argv);
+
+	if (mount(NULL, "/", NULL, MS_REMOUNT | MS_RDONLY | MS_BIND, NULL)) {
+		pr_perror("mount");
+		return 1;
+	}
+
+	test_daemon();
+	test_waitsig();
+
```

Comment:
should we check that the mount is still read-only?

Reply:
zdtm already checks it for all the tests with mntns, please see https://github.com/checkpoint-restore/criu/pull/2637#issuecomment-2771652435

Reply:
I had to read comments:). Let's add a comment in the test to explain why it doesn't check anything.

Reply:
done